doc/content/lorawan/db-dbm-dbi-dbd/_index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "dB, dBm, dBi, and dBd"
+section: Additional Information
+description: ""
+weight:
+---
+
+In this chapter we learn about some units in brief that are used to measure the performance of transceivers (gateways and end devices) and antennas.
+
+## dB (decibel)
+
+The decibel can be used to express the ratio of two physical quantities such as power, sound intensity, sound pressure, voltage, and current on a logarithmic scale. In LoRaWAN we use decibel to express the ratio between two power levels usually given in Watt (W) or milliwatt (mW).
+
+The power ratio can be expressed in decibels using the formula, 
+
+**dB = 10 log<sub>10</sub> (P<sub>out</sub>/P<sub>in</sub>)**
+
+where P<sub>out</sub> is the output power and P<sub>in</sub> is the input power.
+
+{{< note "When we are dealing with the power levels we use 10log units." />}}
+
+For example, if an amplifier turns a 1 W signal into a 1000 W signal, its power ratio can be expressed as,
+
+dB = 10 log<sub>10</sub> (P<sub>out</sub>/P<sub>in</sub>) = 10 log<sub>10</sub> (1000/1) = 30 dB
+
+Decibel doesn’t provide an absolute value. By looking at the decibel value you can’t say the input and output power of a device or cable etc, but you can say whether it offers a gain or a loss.
+
+A power ratio greater than 0 dB is treated as a gain. For example, if an amplifier turns a 2 W signal into a 10 W signal, the power ratio is,
+
+10 log<sub>10</sub> P<sub>out</sub>/P<sub>in</sub>  = 10 log<sub>10</sub> 10/2   = 10 log<sub>10</sub> 5  = 6.9 dB (gain)
```

Comment:
This will read better if we add braces around the term for which the log is taken. ie. 10 log<sub>10</sub>(P<sub>out</sub>/P<sub>in</sub>). Otherwise it is a little difficult to read and ambiguous. Without braces it could mean [ 10 log<sub>10</sub>(P<sub>out</sub>) ] / P<sub>in</sub>